rocketpool-cli/node/claim-rewards.go

```diff
@@ -75,7 +75,7 @@ func nodeClaimRewards(c *cli.Context) error {
 		// Download the files
 		for _, missingInterval := range missingIntervals {
 			fmt.Printf("Downloading interval %d file... ", missingInterval.Index)
-			err := rprewards.DownloadRewardsFile(cfg, missingInterval.Index, missingInterval.CID, false)
+			err := rprewards.DownloadRewardsFile(cfg, missingInterval.Index, missingInterval.CID, missingInterval.MerkleRoot, false)
```

Comment:
kind of feels like this should be `missingInterval.DownloadRewardsFile(cfg, false)`

missingInterval is part of `rprewards` anyway.